plugin.js

```diff
@@ -1,9 +1,9 @@
-UPLOADCARE_CROP = true;
+UPLOADCARE_CROP = '';
 UPLOADCARE_AUTOSTORE = true;
```

Comment:
and `UPLOADCARE_CROP` could be made configurable. see https://github.com/uploadcare/uploadcare-redactor/blob/master/uploadcare.js